<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>singleplay</class>
 <widget class="QWidget" name="singleplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>singleplay</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="rounds">
     <item>
      <widget class="QWidget" name="LattersRound" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="finalScore" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="textScore">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="nextRound">
             <property name="text">
              <string>Далее</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="charWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="charLayout"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="answerWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="answerLayout">
           <item>
            <widget class="QLabel" name="textAnswer">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="charChoiser_2" native="true">
          <layout class="QHBoxLayout" name="charChoiser">
           <property name="leftMargin">
            <number>11</number>
           </property>
           <property name="topMargin">
            <number>11</number>
           </property>
           <property name="rightMargin">
            <number>11</number>
           </property>
           <property name="bottomMargin">
            <number>11</number>
           </property>
           <item>
            <widget class="QPushButton" name="vowel">
             <property name="text">
              <string>Гласная</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="consonant">
             <property name="text">
              <string>Согласная</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="controlButtons" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="skip">
             <property name="text">
              <string>Пропустить</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="answer">
             <property name="text">
              <string>Ответить</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="NumbersRound" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="numberWidget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4"/>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="answerWidget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="textAnswer_2">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="controlButtons_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>100</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Большие</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="countBigNumbers">
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="maximum">
                  <number>4</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Expanding</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>100</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Маленькие</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="countSmallNumbers">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::NoFocus</enum>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                 <property name="keyboardTracking">
                  <bool>true</bool>
                 </property>
                 <property name="minimum">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <number>6</number>
                 </property>
                 <property name="value">
                  <number>6</number>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="accept">
             <property name="text">
              <string>Принять</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="exit" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>240</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="exit_2">
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>vowel</sender>
   <signal>clicked()</signal>
   <receiver>singleplay</receiver>
   <slot>createVowel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>consonant</sender>
   <signal>clicked()</signal>
   <receiver>singleplay</receiver>
   <slot>createConsonant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_2</sender>
   <signal>clicked()</signal>
   <receiver>singleplay</receiver>
   <slot>endGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createVowel()</slot>
  <slot>createConsonant()</slot>
  <slot>endGame()</slot>
 </slots>
</ui>
